data/sql/updates/pending_db_world/rev_1667348648920858600.sql

```diff
@@ -0,0 +1,4 @@
+--
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1` = 5 WHERE `creature_id` IN (674, 675, 676, 677, 1094, 1095, 1096, 1097, 4260, 4723);
+UPDATE `creature_onkill_reputation` SET `MaxStanding1` = 5 WHERE `creature_id` IN  (921);
```

Comment:
Being informed it is not working as intended.
https://github.com/azerothcore/azerothcore-wotlk/pull/13644#issuecomment-1304866420